bin/install.sh

```diff
@@ -38,13 +38,13 @@ echo HOST_NAME=$hostName >> $envFile
 sed -n '/WAFFLE_BASE_URL/!p' $envFile > tmp.list && mv tmp.list $envFile
 echo WAFFLE_BASE_URL="https://${hostName}" >> $envFile
 sed -n '/WAFFLE_API_BASE_URL/!p' $envFile > tmp.list && mv tmp.list $envFile
-echo WAFFLE_API_BASE_URL="https://api.${hostName}" >> $envFile
+echo WAFFLE_API_BASE_URL="https://${hostName}/api" >> $envFile
 sed -n '/WAFFLE_HOOKS_SERVICE_URI/!p' $envFile > tmp.list && mv tmp.list $envFile
-echo WAFFLE_HOOKS_SERVICE_URI="https://hooks.${hostName}" >> $envFile
+echo WAFFLE_HOOKS_SERVICE_URI="https://${hostName}/hooks" >> $envFile
 sed -n '/RALLY_INTEGRATION_BASE_URL/!p' $envFile > tmp.list && mv tmp.list $envFile
-echo RALLY_INTEGRATION_BASE_URL="https://rally.${hostName}" >> $envFile
+echo RALLY_INTEGRATION_BASE_URL="https://${hostName}/integrations/rally" >> $envFile
 sed -n '/POXA_HOST/!p' $envFile > tmp.list && mv tmp.list $envFile
-echo POXA_HOST="poxa.${hostName}" >> $envFile
+echo POXA_HOST="${hostName}" >> $envFile
```

Comment:
solution: let's forward requests to port `8000` to `poxa`, but keep `/app` and `/apps` how they are, so opening port 8000 to the wild is optional.